dashboard/test/integration/curriculum_docs_test.rb

```diff
@@ -131,6 +133,20 @@ def setup
       end
       assert_response :success
     end
+
+    test "class show should cache all queries" do
+      assert_cached_queries(0) do
```

Comment:
I don't understand the assertion here, the test is labeled "should cache all queries" but `assert_cached_queries(0)` makes it seem like there are 0 cached queries

Reply:
Oof yeah it is ambiguous. What it [actually does](https://github.com/code-dot-org/code-dot-org/blob/98f43456b6b72b1d140ec5a02335bcc40d5fec6d/dashboard/test/testing/capture_queries.rb#L32) is run the same block twice and assert the number of queries after the second time. So really, it should be something like `assert_queries_after_caching`.

Reply:
Ahh got it